clang/test/SemaCXX/noescape-attr.cpp

```diff
@@ -3,5 +3,11 @@
 template<typename T>
 void test1(T __attribute__((noescape)) arr, int size);
 
-// expected-warning@+1 {{'noescape' attribute only applies to pointer arguments}}
-void test2(int __attribute__((noescape)) arr, int size);
```

Comment:
I'm guessing this is also the case in the original patch, but this diagnostic looks off to me. `0 is invalid` doesn't really mean anything. We should fix this.

CC @Xazax-hun

Reply:
Patch up: https://github.com/swiftlang/llvm-project/pull/10747